core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -188,7 +188,7 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
       }
     }
 
-    def abortActiveWorkflows(submissionId: UUID) = {
+    def abortActiveWorkflows(submissionId: UUID): Future[Seq[Future[(Option[String], Try[ExecutionServiceStatus])]]] = {
```

Comment:
I saw that and had the same thoughts. It's probably part of why we send a _lot_ of /abort requests to Cromwell. We could traverse and flatten, but that'd have us blocking on the HTTP responses here. I don't actually think it's super terrible that these execute async - we might send a few more of them than we need to but it'll clear up eventually.